### PR TITLE
Fix duplicate ID issue

### PR DESCRIPTION
**Pull request by chetan51**
Fixes https://github.com/numenta/nupic/issues/2467.

### CODE REVIEW

**Comment by chetan51** [comment]
@scottpurdy 
cc: @rhyolight

File: .gitignore [hunk 1]
```diff
@@ -34,6 +34,7 @@ savedmodels/
 /*.please_wait
 /auto_specials*.csv
 /weighted_mean*.csv
+/agg_gap_hours_24*.csv
```
Comment (scottpurdy) [inline .gitignore]: What's this?

Comment (chetan51) [inline .gitignore]: A file generated by one of the integration tests. Even before this change it was being generated, but not gitignored.

**Comment by scottpurdy** [comment]
Rather than closing #2556 with this, can you pick one of the sub-issues that is definitely resolved by this and then we can monitor for a bit before closing #2556.

**Comment by scottpurdy** [comment]
:+1:

**Comment by chetan51** [comment]
@scottpurdy Done.

**Pull request merged**